tools/flaky-test-retryer/main.go

```diff
@@ -34,13 +34,14 @@ const (
 func main() {
 	serviceAccount := flag.String("service-account", os.Getenv("GOOGLE_APPLICATION_CREDENTIALS"), "JSON key file for GCS service account")
 	githubAccount := flag.String("github-account", "", "Token file for Github authentication")
+	dryrun := flag.Bool("dry-run", false, "dry run switch")
 	flag.Parse()
 
 	if err := InitLogParser(*serviceAccount); nil != err {
 		log.Fatalf("Failed authenticating GCS: '%v'", err)
 	}
 
-	handler, err := NewHandlerClient(*githubAccount)
+	handler, err := NewHandlerClient(*githubAccount, *dryrun)
```

Comment:
I feel like this can be done slightly different, like https://github.com/knative/pkg/blob/e4bc08cc8deda3eac40a8d53984d09c9d0e1ad48/test/e2e_flags.go#L48

Reply:
moved all flags to that format, not necessary but figured I'd do it for consistency